packages/rwa-oracles/scripts/deploy.sh

```
@@ -0,0 +1,41 @@
+if [ -z "$RPC_URL" ]; then
+  echo "Error: $RPC_URL_VAR not set in environment"
+  exit 1
+fi
+
+echo "Deploying to $NETWORK..."
+
+# Run forge script and capture output
+# We use --broadcast to actually send txs
+# We need DEPLOYER_PRIVATE_KEY in env
+OUTPUT=$(forge script scripts/DeployRwaOracle.s.sol:DeployRwaOracle 
+  --rpc-url $RPC_URL 
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)